### ISSUE TITLE
about Shrec2018_TripletCenterLoss.pytorch

### ISSUE THREAD

**Issue by jiangyanjavawin** [xlliu7/BMN-Boundary-Matching-Network#2]
hello,thanks for your code about Shrec2018_TripletCenterLoss.pytorch,it's very good,but when i use code with my dataset.py,I have some issues,would you mind share your dataset.sk_dataset as sk_dataset  and dataset.sh_views_dataset as sh_views_dataset ,thanks very much.
ps:my error is about TCL,at the localtion of loss.backward()
The expanded size of the tensor (1) must match the existing size (16) at non-singleton dimension 1. at /pytorch/torch/lib/THC/generic/THCTensor.c:323
If you have any idea,please tell me,thank you very much.
